pkg/proxy/backup.go

```diff
@@ -241,7 +241,29 @@ func (p *Proxy) BackupVolumeGet(ctx context.Context, req *rpc.EngineBackupVolume
 		Volume: &rpc.EngineBackupVolumeInfo{},
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.